### ISSUE TITLE
Add spacer between map scale -value and -unit for better readability

### ISSUE THREAD

**Issue by sirtet** [openlayers/ol3#3116]
The current default, eg. *100km* is very hard to read. *100 km* would be much better.

See the example:
http://openlayers.org/en/v3.1.1/examples/scale-line.html
In contrast to Google maps (scale at bottom right)
https://maps.google.com/

The OL3 default scale is very hard to read, with the __value__ and __unit__ immediately connected.
Maybe also add a class, so they can be styled differently?

**Comment by tsauerwein** [comment]
You are right, a space would really improve the readability. Do you want to work on a PR for this? This would be a good opportunity to get started, please see: https://github.com/openlayers/ol3/blob/master/CONTRIBUTING.md

**Comment by sirtet** [comment]
I would like to, but i am short on time.
So anyone who wants, go ahead...

**Comment by sirtet** [comment]
Just added a simple space.
Thought about instead (or better additionally) adding a span, to allow different styling between value and unit:
```javascript
-  var html = count + suffix;
+  var html = count + ' <span id="scale-unit">' + suffix + '</span>';
```

but someone once said, code only if it's needed by someone.

**Comment by tsauerwein** [comment]
Closed with #3124
